### PR TITLE
Avoid assertion when bound `id` is provided to tagless component

### PR DESCRIPTION
**Pull request by HeroicEric**
See https://github.com/emberjs/ember.js/pull/14382

/cc @rwjblue

### CODE REVIEW

**Comment by HeroicEric** [comment]
Updated with suggestion from https://github.com/emberjs/ember.js/pull/14382#discussion_r81139229

File: packages/ember-glimmer/tests/integration/components/fragment-components-test.js [hunk 1]
```diff
@@ -165,6 +165,27 @@ moduleFor('Components test: fragment components', class extends RenderingTest {
...
+      tagName: ''
+    });
+
+    this.registerComponent('foo-bar', { ComponentClass: FooBarComponent, template });
+
+    this.render(`{{#foo-bar id=fooBarId}}{{/foo-bar}}`, { fooBarId: 'baz' });
+
+    this.assertText('baz');
+
+    this.runTask(() => set(this.context, 'fooBarId', 'qux'));
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/components/fragment-components-test.js]: Can you do a noop rerender before this mutation step?


This should do the trick:

```
this.assertStableRerender();
```

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: packages/ember-glimmer/tests/integration/components/fragment-components-test.js (hunk 1 above)
Comment (HeroicEric) [inline packages/ember-glimmer/tests/integration/components/fragment-components-test.js]: done

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Thanks @HeroicEric!

**Pull request merged**